wordpress.org/public_html/wp-content/themes/pub/wporg-main/front-page.php

```diff
@@ -27,20 +27,17 @@
 $swag_class = $showcase ? 'col-4' : 'col-2';
 $user_class = $showcase ? 'col-12' : 'col-2';
 
-$wp20_url = 'https://wp20.wordpress.net/';
-if ( is_object( $rosetta ) && isset( $rosetta->locale ) ) {
-	$wp20_url .= '?locale=' . $rosetta->locale;
-}
+$banner_url = 'https://wordpress.org/state-of-the-word/';
```

Comment:
The State of the Word page isn't available in other locales right now (it wasn't pushed to Rosetta sites, but I can't find any context for why not), so this can be removed — done in https://github.com/WordPress/wordpress.org/pull/179/commits/e2e9deca76c231870d27ae4ed3d6a7c5da2eea99